plugins/device-injector/README.md

```diff
@@ -26,6 +26,33 @@ The annotation syntax for device injection is
 
 `file_mode`, `uid` and `gid` can be omitted, the rest are mandatory.
 
+### CDI Device Annotations
+
+Devices are annotated using the `cdi-devices.nri.io` annotation key prefix.
```

Comment:
It seems that no additional attributes are needed here; wouldn't using a comma-separated format be more convenient?

Reply:
Would definitely be possible but it is a bit subjective which one is more convenient. I prefer keeping device injection and CDI device injection syntactically as close to each other as possible here. Also, the CRI protocol uses a `struct { Name string }` for passing CDI devices to inject and this NRI patch set follows suit. That allows for future extensions with additional per-device attributes. The current strightforward struct-marshalled-to-yaml annotation syntax can be trivially extended to support such extensions (and it is also syntactically forward-compatible). A comma-separated syntax, however, would need to be abandoned in favor of the current syntax if ever such extensions were introduced.